examples/concourse-terraform/cepler.yml

```diff
@@ -0,0 +1,8 @@
+environments:
+  staging:
+    latest:
+    - examples/concourse-terraform/terraform/*.tf
+  production:
+    latest:
+    - examples/concourse-terraform/terraform/*.tf
```

Comment:
Could we have a more realistic example? The examples should act as documentation. So at a minimum there should be 3 files to show how things could work.
Perhaps `main.tf` `staging.tfvars` and `production.tfvars` or something like that?

Reply:
updated to use tfvars